app/src/main/contrib/components/Responsibilities/Overview/List/TemplateLinks.tsx

```diff
@@ -130,8 +130,18 @@ export class TemplateLink extends React.Component<TemplateLinkProps, undefined>
 
         if (settings.isApplicantTouchstone(this.props.touchstoneId)) {
             return <div>
+                <div>
                     <a key={`central_burden_template_${disease.id}-generic.csv`}
-                       href={`${templatePath}central_burden_template_${disease.id}-generic.csv`}>{disease.name}</a>
+                       href={`${templatePath}central_burden_template_${disease.id}-generic.csv`}>{disease.name}
+                        { settings.is201804ApplicantTouchstone(this.props.touchstoneId) && " - central"}</a>
+                </div>
+                { settings.is201804ApplicantTouchstone(this.props.touchstoneId) &&
```

Comment:
I don't mind the `{ predicate && renderLogic }` approach for one liners, as above, but for this maybe put the render logic into a separate function?